.github/workflows/cockpit-container-update.yml

```diff
@@ -0,0 +1,31 @@
+name: cockpit-container-update
+on:
+  schedule:
+    - cron: '0 2 * * 4'
```

Comment:
I'd actually like to do this daily. But, does the  bot check if it already opened a PR? If not, then let's keep weekly for now.